tests/impl.cpp

```diff
@@ -6899,7 +6915,24 @@ result_t test_mm_alignr_epi8(const SSE2NEONTestImpl &impl, uint32_t iter)
 
     __m128i a = load_m128i(_a);
     __m128i b = load_m128i(_b);
-    __m128i ret = _mm_alignr_epi8(a, b, shift);
+    __m128i ret;
+    switch (iter % 5) {
```

Comment:
Ditto. Rewrite as
```c
__m128i ret = _mm_alignr_epi8(a, b, (iter % 5) << 3);
```

Reply:
Done.

Reply:
The code can not be rewritten since the third parameter should be an immediate value, which means the value should be determined at compile time.